org.jupyter.JupyterLab.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: org.jupyter.JupyterLab
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it support wayland?

Reply:
It doesn't; I've removed the permission.